services/pg-backup/docker-compose.yml

```diff
@@ -1,14 +1,20 @@
 version: '3.7'
 services:
-  dbbackup:
-    image: justb4/pgbackup:10
+  SERVICE_NAME_PARAMETRIZED:
+    image: kartoza/pg-backup:14-3.3@sha256:4d1867707b1da879324199fb3c0d542122113bfc50cd2418764efb8f4309d6c1
     volumes:
-    - /var/run/docker.sock:/var/run/docker.sock
-    - /tank:/backup
+      - /tank/${MACHINE_FQDN}:/backups
```

Comment:
What happens if the folder /tank/${MACHINE_FQDN} doesnt exist? Will it be created or will the service fail?

Reply:
Service fail - it need to be created each time we have a new deployement, or updated each time we change this variable

Reply:
OK this sounds good to me, it's better to fail than to have a weird behaviour.